__tests__/app.test.js

```diff
@@ -0,0 +1,206 @@
+        expect(review).toHaveProperty("votes", expect.any(Number));
+        expect(review).toHaveProperty("category", expect.any(String));
+        expect(review).toHaveProperty("review_body", expect.any(String));
+        expect(review).toHaveProperty("owner", expect.any(String));
+        expect(review).toHaveProperty("created_at", expect.any(String));
+      });
+  });
+  it("if review with id = two requested should respond with a review object with id two, with corresponding id and properties", () => {
+    return request(app)
+      .get("/api/reviews/2")
+      .expect(200)
+      .then(({ body }) => {
+        const { review } = body;
+        expect(review.review_id).toBe(2);
+        expect(review).toHaveProperty("title", expect.any(String));
+        expect(review).toHaveProperty("designer", expect.any(String));
+        expect(review).toHaveProperty("review_img_url", expect.any(String));
+        expect(review).toHaveProperty("votes", expect.any(Number));
+        expect(review).toHaveProperty("category", expect.any(String));
+        expect(review).toHaveProperty("review_body", expect.any(String));
+        expect(review).toHaveProperty("owner", expect.any(String));
+        expect(review).toHaveProperty("created_at", expect.any(String));
+      });
+  });
+  it("if the request is for an id which doesn't exist, responds with status 404", () => {
+    return request(app)
+      .get("/api/reviews/100")
+      .expect(404)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toBe("404: No user found for user_id: 100");
+      });
+  });
+  it("if the request is ill-formed, responds with status 400", () => {
+    return request(app)
+      .get("/api/reviews/cats")
+      .expect(400)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toBe("400: ill-formed request");
+      });
+  });
+});
+
+describe("GET /api/reviews/:review_id/comments", () => {
+  it("if comments from review with id = two are requested should respond with an array of comment objects with corresponding reviews_id and other properties", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        for (let comment of comments) {
+          expect(comment.review_id).toBe(2);
+          expect(comment).toHaveProperty("body", expect.any(String));
+          expect(comment).toHaveProperty("author", expect.any(String));
+          expect(comment).toHaveProperty("comment_id", expect.any(Number));
+          expect(comment).toHaveProperty("votes", expect.any(Number));
+          expect(comment).toHaveProperty("created_at", expect.any(String));
+        }
+      });
+  });
+  it("if comments from review with id = three are requested should respond with an array of comment objects with corresponding reviews_id and other properties", () => {
+    return request(app)
```

Comment:
Can we add a status 200 test that if it has a valid id but has no comments it responds with an empty array of comments.